setup.py

```diff
@@ -28,7 +28,6 @@
         'Intended Audience :: Developers',
         'Topic :: Software Development :: Libraries',
         'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 3.7',
```

Comment:
I'd like to move dropping python 3.7 and adding mypy in separate PR(s) and merge soon, then rebase to that.  I may give it a try.

Reply:
I'll likely be busy the rest of the week, but can do it afterwards if you haven't gotten around to it.